.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,72 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+name: "CodeQL"
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master]
+  schedule:
+    - cron: '0 8 * * 3'
```

Could we add a comment, I always have to look this up and I would imagine others do as well...
```suggestion
    # Run every Wednesday at 8 am 
    - cron: '0 8 * * 3'
```
I can't find anything on which timezone this would be but since it's only once a week (besides PRs), it probably doesn't matter much.